custom_components/tuya_local/devices/vta+_fan_heater

```diff
@@ -0,0 +1,79 @@
+name: Fan heater
+products:
+  - id: ebcb3fda096d331d27h8yc
```

Comment:
this looks like a device id.
product id is now output in the log when you use the cloud config flow.